src/plugins/console/server/lib/spec_definitions/js/mappings.ts

```diff
@@ -159,28 +157,25 @@ export const mappings = (specService: SpecDefinitionsService) => {
 
           // dates
           format: {
-            __one_of: _.flatten([
-              _.map(
-                [
-                  'date',
-                  'date_time',
-                  'date_time_no_millis',
-                  'ordinal_date',
-                  'ordinal_date_time',
-                  'ordinal_date_time_no_millis',
-                  'time',
-                  'time_no_millis',
-                  't_time',
-                  't_time_no_millis',
-                  'week_date',
-                  'week_date_time',
-                  'week_date_time_no_millis',
-                ],
-                function (s) {
-                  return ['basic_' + s, 'strict_' + s];
-                }
-              ),
-              [
+            __one_of: [
+              ...[
```

Comment:
started showing a type error

Reply:
@jloleysens Could you take a look at this please? Looks like this is the only file that's owned by ES UI.